src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,19 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @OneToMany
```

Comment:
I think ManyToMany is better. But as it passed tests it might be ok.

Reply:
Agree, one actor can act in many Movies, and one movie has multiple actors, so that is a many-to-may relation